ccmbioinfo/lfs
# CARDS-1642: Make all subject types project-independent

To test:
- build
- start with each of the supported projects (`proms`, `lfs`, `care`, `kids`) as well as test mode (`--test`), make sure the right subject types and the right questionnaires are present
- delete the database (`rm -rf .cards-data`) between each tested project
